frontend/src/index.js

```diff
@@ -33,4 +33,4 @@ ReactDOM.render(
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
-serviceWorker.unregister();
+serviceWorker.register();
```

Comment:
Will this also cache the dataset which is used for the graphs? If so, we need some way of invalidating it manually / based on the action of loading new data into the API / bucket.

Reply:
It's only caching static assets.